extensions/vertx/deployment/pom.xml

```diff
@@ -61,11 +53,6 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
```

Comment:
Wasn't that one used?

Reply:
We had to move the related tests elsewhere because of a cyclic dependency...

Reply:
And in fact, it was the only one ;-)